src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+const spiderRect = spider.getBoundingClientRect();
+const spiderWidth = spiderRect.width;
+const spiderHeight = spiderRect.height;
+
+wall.addEventListener('click', e => {
+  if (e.target === wall) {
+    const wallRect = wall.getBoundingClientRect();
+    const clickX = e.clientX - wallRect.left;
+    const clickY = e.clientY - wallRect.top;
+
+    const spiderCenterX = clickX - spiderWidth + 15;
+    const spiderCenterY = clickY - spiderHeight + 10;
```

Comment:
Now values 10 and 15 looks like 'magical' and it hard to understand where did we get it